src/main/java/com/example/b2b_opportunities/Entity/Seniority.java

```diff
@@ -0,0 +1,27 @@
+package com.example.b2b_opportunities.Entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@RequiredArgsConstructor
+@Table(name = "seniorities")
+public class Seniority {
+    @Id
+    @Column(name = "id", nullable = false)
+    private String id;
```

Comment:
I saw that the FE team sent the IDs as strings. But I think it is better to have them as numbers. Usually for PK is better to use numbers (you can check the top 2 responses) - https://stackoverflow.com/questions/3162202/sql-primary-key-integer-vs-varchar 

If we need those S1, T1 keys, we can have another column for this. We have to discuss this with the FE team - can you write a question in the ClickUp about it?

Reply:
This is fixed now.